k8s/namespaces/rd/rd-professional-api/perftest.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: rd-professional-api
+spec:
+  values:
+    java:
+      environment:
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.perftest.platform.hmcts.net
+        IDAM_URL: https://idam-api.perftest.platform.hmcts.net
+        OPEN_ID_API_BASE_URI: https://idam-web-public.perftest.platform.hmcts.net/o
```

Comment:
https://github.com/hmcts/rd-professional-api/blob/master/charts/rd-professional-api/values.yaml#L12

you shouldn't need any of these patches except in prod, you the environment in your chart, do you need IDAM_URL and AUTH_IDAM_CLIENT_BASEURL? they aren't specified in your chart currently